gatsby-node.js

```diff
@@ -97,6 +106,19 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
       }
     })
   }
+
+  if (markdownDocs.nodes.length) {
```

Comment:
ok, so if we have the same path for a page created in WordPress and markdown, will this override the page created by WordPress? 

That seems to be what I'm seeing, eh? We do have a `faqs` docs entry in both WP and Markdown, but from what I can tell, there's only one page Gatsby is creating and it's the markdown one.

Reply:
I played around with order, creating a page from MarkDown before the allContent nodes and the last page created with the same uri was the winner.  I see https://www.wpgraphql.com/docs/faqs/ which does exist in the WP content.